### DISCUSSION TITLE
Terms of Use

### DISCUSSION THREAD

**Discussion by andre-de-lorde** [Tyrrrz/DiscordChatExporter#1198] [General]
Um, what? No.
Why do I need to do that just to run a script???

**Comment by Technetium1** [comment]
For the same reason you agree to terms of use for all the other software you want to use. Complaining won't change anything.

**Comment by andre-de-lorde** [comment]
normal terms of service don't require you to re/denounce a country or military to use it.

**Reply by Technetium1** [reply]
https://github.com/search?q=condemn+Russia&type=code It's definitely not abnormal. Look how many results.

**Reply by andre-de-lorde** [reply]
Half of those are "code" containing the words condemn or Russia. If you strict search there is only 1.3k and even then, most of them are News. On the first page alone:
1 is someone personally condemning them, not ToS
15 are a news archive
2 are another project by Tyrrrz containing the same weird ToS
2 are training data for an AI

If you search more accurately for "You **condemn Russia and its military aggression against Ukraine**" there are only 106 results. Of the 106 results only 13 out of them are unique projects—with 2 of them being advent of code— that aren't just forks of projects containing this wacky ToS or made by Tyrrrz. Making it just 10.38% of the projects where you **strict search** for ToS like this, 12.26% if you're desperate, So no, this IS abnormal.

**Reply by 96-LB** [reply]
This is such a silly hill to die on. Why does it matter whether it's abnormal or not? Tyrrrz's countless hours of work for this project are being provided to you, free of charge, and you've chosen to complain that his Terms of Service ask you not to support the country bombing his home and threatening his life. These terms aren't even enforced. You are not entitled to his work—if it bothers you so much on a moral level to agree to these terms, you may exercise your free will to not use DiscordChatExporter. Please take a step back and realize how ridiculous it is to be so hostile towards the ToS of a free, accessible project that *you* chose to make a part of your life.
